scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -159,6 +161,21 @@ namespace <%= namespace %> {
         }
 
         /// <summary>
+        /// Launches an external browser with the Cart's web checkout link loaded. This can be used to perform
+        /// a cart checkout when no native web view is available, such as on desktop, or from the Unity editor.
+        /// </summary>
+        /// <param name="success">called when the game has regained focus and the checkout was successful.</param>
+        /// <param name="cancelled">called when the game has regained focus but the checkout has not completed.</param>
+        /// <param name="failure">called when the game has regained focused and an error has occured during checkout.</param>
+        public void CheckoutWithExternalBrowser(CheckoutSuccessCallback success, CheckoutCancelCallback cancelled, CheckoutFailureCallback failure) {
```

Comment:
@dengjeffrey @sleroux Question for you guys, thinking in terms of https://github.com/Shopify/unity-buy-sdk/issues/291, it seems there are a few ways we would go with adding in the functionality around `Application.OpenURL`

Option 1. We make this method apply only to standalone and editor builds, which entails making the name something like, CheckoutWithDesktopBrowser, and we use ifdefs to only define that type of checkout for standalone.

Option 2. There isn't any technical reason (AFAIK) that we couldn't call `Application.OpenURL` in a mobile context, so we could keep this as a separate method, but allow it to be called on mobile. If that was the case, I think we drop the Desktop part of the name, and call it ExternalCheckout or similar, and do not use ifdefs to prevent it from being defined for mobile.

Option 3. We don't add a new method, but instead we add logic to `CheckoutWithNativeWebView` in order to use `Application.OpenUrl` when not on a mobile device. This would mean the name of the method no longer makes sense, so we'd want to change it to perhaps just `CheckoutWithWebView`. One detail here, on iOS we do launch a "true" web view (in that you can't just browse away from it) but on android we are basically just launching an external browser anyways (although it does feel a but more like a web view).

So, there are some further questions about if and how we handle the callbacks on the function we choose to implement, but I think that's our first decision to make.